manifests/deployment.yml

```diff
@@ -1,67 +0,0 @@
-# Grant OPA/kube-mgmt read-only access to resources. This lets kube-mgmt
-# list configmaps to be loaded into OPA as policies.
-kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: opa-viewer
-roleRef:
-  kind: ClusterRole
-  name: view
-  apiGroup: rbac.authorization.k8s.io
-subjects:
-- kind: Group
-  name: system:serviceaccounts:opa
-  apiGroup: rbac.authorization.k8s.io
----
-# Define role for OPA/kube-mgmt to update configmaps with policy status.
-kind: Role
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  namespace: opa
-  name: configmap-modifier
-rules:
-- apiGroups: [""]
-  resources: ["configmaps"]
-  verbs: ["update", "patch"]
----
-# Grant OPA/kube-mgmt role defined above.
-kind: RoleBinding
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  namespace: opa
-  name: opa-configmap-modifier
-roleRef:
-  kind: Role
-  name: configmap-modifier
-  apiGroup: rbac.authorization.k8s.io
-subjects:
-- kind: Group
-  name: system:serviceaccounts:opa
-  apiGroup: rbac.authorization.k8s.io
----
-# Deploy OPA and kube-mgmt.
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: opa
-spec:
-  selector:
-    matchLabels:
-      app: opa
-  replicas: 1
-  template:
-    metadata:
-      labels:
-        app: opa
-    spec:
-      containers:
-      - name: opa
-        image: openpolicyagent/opa
-        args:
-        - "run"
-        - "--server"
-        ports:
-        - name: http
-          containerPort: 8181
-      - name: kube-mgmt
-        image: openpolicyagent/kube-mgmt:0.12.1
```

Comment:
A sample manifest to deploy `kube-mgmt` without helm would be useful to have.

Reply:
Replied above